pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -52,6 +52,7 @@ public enum ControllerGauge implements AbstractMetrics.Gauge {
   OFFLINE_TABLE_COUNT("TableCount", true),
   DISABLED_TABLE_COUNT("TableCount", true),
   PERIODIC_TASK_NUM_TABLES_PROCESSED("PeriodicTaskNumTablesProcessed", true),
+  MINION_TASK_METADATA_LAST_UPDATE_TIME_MS("MinionTaskMetadataLastUpdateTimeMs", true),
```

Comment:
Can we think of normalizing the metrics? Reporting the raw timestamp value will require ppl to re-interpret that to the human readable format so it's going to be a bit hard to use.

For example, the same information can be defined as `the time since last update in hours (or days)`. This will show as low number in regular time and it will increase if there's no update at all for a long time. Once we make some progress, it will go down to 0 again.

Reply:
Had an offline discussion with @snleee .

Although it is fairly easy to perform calculation in PromQL (e.g., `now() - last update time` will be the time since the last update time), it may still be more straight forward if we use normalized metrics.

Therefore, we will update this metric to be `the time since the last update time`